<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>VariableEditor</class>
 <widget class="QWidget" name="VariableEditor">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>697</width>
    <height>443</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Редактор аргументов</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QWidget" name="nameWdg" native="true">
     <layout class="QHBoxLayout" name="nameLay">
      <item>
       <widget class="QLabel" name="nameLbl">
        <property name="text">
         <string>Название:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="nameEdt"/>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="iContentHbox">
     <item>
      <widget class="QGroupBox" name="iVarGbox">
       <property name="title">
        <string>Переменные</string>
       </property>
       <layout class="QVBoxLayout" name="ivarsVbox">
        <item>
         <widget class="QListWidget" name="varLst">
          <property name="editTriggers">
           <set>QAbstractItemView::NoEditTriggers</set>
          </property>
          <property name="showDropIndicator" stdset="0">
           <bool>false</bool>
          </property>
          <property name="alternatingRowColors">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="iVarControlsHbox">
          <item>
           <widget class="QPushButton" name="addVarBtn">
            <property name="text">
             <string>Добавить</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="removeVarBtn">
            <property name="text">
             <string>Удалить</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="iVarControlsSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="iChGbox">
       <property name="title">
        <string>Каналы</string>
       </property>
       <layout class="QVBoxLayout" name="iChVbox">
        <item>
         <widget class="QTableWidget" name="chTbl">
          <property name="editTriggers">
           <set>QAbstractItemView::NoEditTriggers</set>
          </property>
          <property name="tabKeyNavigation">
           <bool>false</bool>
          </property>
          <property name="showDropIndicator" stdset="0">
           <bool>false</bool>
          </property>
          <property name="dragDropOverwriteMode">
           <bool>false</bool>
          </property>
          <property name="alternatingRowColors">
           <bool>true</bool>
          </property>
          <property name="selectionBehavior">
           <enum>QAbstractItemView::SelectRows</enum>
          </property>
          <attribute name="verticalHeaderVisible">
           <bool>false</bool>
          </attribute>
          <column>
           <property name="text">
            <string>Спутник</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Прибор</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Канал</string>
           </property>
          </column>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="iChControlsHbox">
          <item>
           <widget class="QPushButton" name="addChBtn">
            <property name="text">
             <string>Добавить</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="removeChBtn">
            <property name="text">
             <string>Удалить</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="iChControlsSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="iControlsHbox" stretch="2,1">
     <property name="leftMargin">
      <number>6</number>
     </property>
     <property name="rightMargin">
      <number>6</number>
     </property>
     <item>
      <spacer name="iControlsSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="completeBtn">
       <property name="minimumSize">
        <size>
         <width>200</width>
         <height>0</height>
        </size>
       </property>
       <property name="text">
        <string>Завершить редактировние</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>nameEdt</tabstop>
  <tabstop>varLst</tabstop>
  <tabstop>addVarBtn</tabstop>
  <tabstop>removeVarBtn</tabstop>
  <tabstop>chTbl</tabstop>
  <tabstop>addChBtn</tabstop>
  <tabstop>removeChBtn</tabstop>
  <tabstop>completeBtn</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
